src/renderer/components/hotbar/hotbar-remove-command.tsx

```diff
@@ -53,8 +53,13 @@ const NonInjectedHotbarRemoveCommand = observer(({
     } }
     components={{ DropdownIndicator: null, IndicatorSeparator: null }}
     menuIsOpen={true}
-    options={hotbarStore.hotbars}
-    getOptionLabel={hotbar => hotbarStore.getDisplayLabel(hotbar)}
+    options={(
+      hotbarStore.hotbars
+        .map(hotbar => ({
+          value: hotbar,
+          label: hotbarStore.getDisplayLabel(hotbar),
+        }))
+    )}
```

Comment:
Nit: it is not UI-code's responsibility to do something like this.

Reply:
To do what?

Reply:
Compute option labels?

Reply:
To do business logic. There's a lot the be said on this, but resolving this nit for now.

Reply:
I am under the impression that we have different definitions of "business code" and I am pretty sure that it isn't as clear cut as you make it out to be.